meta-mender-qemu/recipes-core/systemd/files/01-logs-to-console.conf

```diff
@@ -0,0 +1,2 @@
+[Journal]
+ForwardToConsole=true
```

Comment:
Won't this be unbearably verbose for a demo environment? I think Mender logs every few seconds when it checks the server.

I can see the value of having this in the tests though, maybe we can do this in the client initialization in the test framework?